src/js/container/login/index.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
```

Comment:
Wherever possible use optional chaining.

a.b.c --> a?.b?.c

So by any chance any object attribute is undefined it won't break the system.